stdlib/_curses.pyi

```diff
@@ -478,9 +478,9 @@ class _CursesWindow:
     @overload
     def insstr(self, y: int, x: int, str: str, attr: int = ...) -> None: ...
     @overload
-    def instr(self, n: int = ...) -> _chtype: ...
+    def instr(self, n: int = ...) -> str: ...
```

Comment:
This overload should return `bytes` too. Both overloads of `instr()` use this return statement: https://github.com/python/cpython/blob/46c8d915715aa2bd4d697482aa051fe974d440e1/Modules/_cursesmodule.c#L1785

```suggestion
    def instr(self, n: int = ...) -> bytes: ...
```